pkgs/by-name/ca/catppuccin-cursors/package.nix

```
@@ -1,36 +1,47 @@
 { lib
 , stdenvNoCC
 , fetchFromGitHub
-, unzip
+, inkscape
+, just
+, xorg
```

```suggestion
, xcursorgen
```

I believe we can bring in `xcursorgen` directly.